src/libsystemd-network/sd-ndisc.c

```diff
@@ -500,33 +509,133 @@ _public_ sd_radv *sd_radv_unref(sd_radv *ra) {
                 sd_radv_prefix_unref(p);
         }
 
+        radv_reset(ra);
+
         sd_radv_detach_event(ra);
         return mfree(ra);
 }
 
+static int radv_send(sd_radv *ra, struct in6_addr *dst,
+                     uint32_t router_lifetime) {
+
+        return -ENOSYS;
+}
+
+static usec_t radv_compute_timeout(usec_t min, usec_t max) {
+
+        return MIN(min, max) + (random_u32() % (MAX(min, max) - MIN(min, max)));
```

Comment:
... drop those MIN, MAX calls here? It'd be easier to read `min + random_u32() % (max - min)`.